frontend/src/features/properties/selector/PropertySelectorFormView.tsx

```diff
@@ -1,72 +1,51 @@
 import * as Styled from 'components/common/styles';
+import {
+  MapCursors,
+  SelectedPropertyContext,
+} from 'components/maps/providers/SelectedPropertyContext';
 import { StyledFormSection } from 'features/mapSideBar/tabs/SectionStyles';
-import { FieldArray, Formik, FormikProps } from 'formik';
-import { IProperty } from 'interfaces';
-import { noop } from 'lodash';
 import * as React from 'react';
 
 import MapClickMonitor from './components/MapClickMonitor';
-import SelectedPropertyHeaderRow from './components/SelectedPropertyHeaderRow';
-import SelectedPropertyRow from './components/SelectedPropertyRow';
-import SelectedPropertyForm from './PropertySelectorSubForm';
-
-export interface IPropertySelectorModel {
-  properties: IProperty[];
-}
+import { IMapProperty } from './models';
+import PropertySelectorSubForm from './PropertySelectorSubForm';
 
 export interface IPropertySelectorFormViewProps {
-  onClickDraftMarker: () => void;
-  onClickAway: () => void;
-  selecting?: boolean;
-  formikRef?: React.MutableRefObject<FormikProps<IPropertySelectorModel>>;
-  properties?: IProperty[];
+  onSelectedProperty: (property: IMapProperty) => void;
 }
 
 const PropertySelectorFormView: React.FunctionComponent<IPropertySelectorFormViewProps> = ({
-  onClickDraftMarker,
-  onClickAway,
-  formikRef,
-  properties,
+  onSelectedProperty,
 }) => {
+  const { setCursor } = React.useContext(SelectedPropertyContext);
```

Comment:
Ideally containers interact with contexts and not form components. That allows for easier segregation of display components and logic components.

Reply:
that's an interesting point. We should talk about it more to standardize the expected behaviour of each part